lib/index.ts

```diff
@@ -39,7 +40,7 @@ class PWMetrics {
     view: false,
     expectations: false,
     json: false,
-    chromeFlags: []
+    chromeFlags: ''
```

Comment:
another thing, I want to add `--headless --no-sandbox --disable-setuid-sandbox` by default in another pr. wdyt?

Reply:
What is the objective? 
Faster overall pwmetrics execution? I'm concerned the reported values will not match what a developer can manually get.
Last time I tried headless lighthouse performance tests the results were quite different from normal execution and not what I found via chrome dev tools with an empty cache.

Reply:
I'm running all env in ubuntu docker container and could not run Chrome without `--headless --no-sandbox --disable-setuid-sandbox`...

@paulirish have either you or team ever experienced different results comparing between headless run and regular one?

Reply:
> @paulirish have either you or team ever experienced different results comparing between headless run and regular one?

--headless used to not support network throttling at all, so the perf numbers were indeed VERY different.
now it DOES. yay!

so the numbers will be closer. it won't be doing some painting/gpu work that headful usually does, but should be roughly comparable.

that said, we haven't switched to using headless for any performance sensitive measurement yet.  just haven't spent the time to confirm the numbers are good enough. :)

Reply:
Ok, so let's just document this case for now, without adding those flags by default.